FC

```diff
@@ -0,0 +1,12 @@
+
+-- Fixed spawntime
+UPDATE `acore_world`.`creature` SET `spawntimesecs`='60' WHERE  `guid`=66716;
```

Comment:
2 Things here:
1. Remove `acore_world`. from the query. Sometimes people use different names for their DBs and this might cause problems.
2. Remove single-quotes from the '60'